src/cmake-tools.ts

```diff
@@ -513,7 +575,11 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
     if (build_retc !== 0) {
       return build_retc;
     }
-    const drv = await this._cmakeDriver;
+
+    const drv = await this.getCMakeDriverInstance();
+    if (!drv) {
+      return -1;
```

Comment:
I think this should be returned as a `Promise`:
```ts
return Promise.resolve(-1);
```

Reply:
Style guide question